.circleci/config.yml

```diff
@@ -22,6 +23,7 @@ references:
   nightly_environment: &nightly_environment
     DOKS_CLUSTER_NAME: codacy-doks-cluster-release
     DO_TF_WORKSPACE: release
+    RELEASE_NAME: codacy-release
```

Comment:
this name doesn't seem to match with the name of the anchor `nightly_environment`. Is this switched?